test/test_compressor.py

```diff
@@ -176,6 +176,13 @@ def _test_compress_to_file(self, filetype, file_size, file_path):
                 assert transfer_event.metadata.pop("hash-algorithm") == "sha1"
             assert getattr(transfer_event, key) == value
 
+        compressed_file_path = os.path.join(
+            self.config["backup_location"], self.config["backup_sites"][self.test_site]["prefix"],
+            "xlog" if filetype == "xlog" else "timeline", file_path.name
+        )
+        # make sure the file was compressed on the expected location
+        assert os.path.exists(compressed_file_path)
```

Comment:
Here I verified we were not uploading missed compressed timeline files during walkup